src/main/resources/application.properties

```diff
@@ -59,11 +61,11 @@ auth.publicEndpoints=/,/favicon.ico,/oauth/start,/oauth/callback
 # Default Database Configuration
 vcap.services.pz-postgres.credentials.jdbc_uri=jdbc:postgresql://localhost:5432/piazza
 # User used for all database read/write interactions
-vcap.services.pz-postgres.credentials.username=piazza
-vcap.services.pz-postgres.credentials.password=piazza
+vcap.services.pz-postgres.credentials.username=postgres
+vcap.services.pz-postgres.credentials.password=admin
 # Used for liquibase management only
-vcap.services.pz-postgres-service-key.credentials.username=piazza
-vcap.services.pz-postgres-service-key.credentials.password=piazza
+vcap.services.pz-postgres-service-key.credentials.username=postgres
+vcap.services.pz-postgres-service-key.credentials.password=admin
```

Comment:
Same comment as above. Lets not change these values. They're all dummy placeholder values to begin with.